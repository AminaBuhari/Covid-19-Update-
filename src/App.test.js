import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import GlobalData from './Components/globalData';

describe('Global Dta', () => {
  describe('Region data', () => {
    it('has Text element', () => {
      render(<GlobalData />);
      const element = screen.getByText('Canada');
      expect(element).toBeInTheDocument();
    });
  });
});

