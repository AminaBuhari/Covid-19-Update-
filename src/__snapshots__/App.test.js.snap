// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots Render country status 1`] = `<app />`;
